task6.rb

```diff
@@ -22,10 +22,25 @@ def *(other_pint)
   end
 end
 
-pint1 = PseudoInt.new(ARGV[0].to_i)
-pint2 = PseudoInt.new(ARGV[1].to_i)
+def is_i?(num)
+  num =~ /\A[-+]?[0-9]+\z/
+end
+
+if is_i?(ARGV[0])
+  pint1 = PseudoInt.new(ARGV[0].to_i)
+else
+  puts "Error: #{ARGV[0]} is not a number"
+  return
+end
+
+if is_i?(ARGV[1])
+  pint2 = PseudoInt.new(ARGV[1].to_i)
+else
+  puts "Error: #{ARGV[1]} is not a number"
+  return
+end
 
 p pint1 + pint2
```

Comment:
I'm running your script with `ruby task6.rb 5 10 sum`
And the output is 4 numbers, which is not correct. Please make the needed corrections.